qcut/docs/issues/caption-implementation/export_bug.md

```
@@ -0,0 +1,145 @@
+✅ [VideoPlayer] Video ready to play: blob:app://./2b28c330-38e2-4f7e-a68c-c251b991e80b
+html2canvas.esm.js:5347 Canvas2D: Multiple readback operations using getImageData are faster with the willReadFrequently attribute set to true. See: https://html.spec.whatwg.org/multipage/canvas.html#concept-canvas-will-read-frequently
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)